types/hapi/test/server/connections.ts

```diff
@@ -11,3 +11,9 @@ server.connection({ port: 8080, labels: 'b' });
 const a = server.select('a');
 
 // a.connections.length === 1
+
+server.connections[0].table().map(({method, path, settings: {description}}) => ({
+    method,
+    path,
+    description
+}));
```

Comment:
Did that and also moved it to the `test/connection/table.ts` file. Didn't see that one previously